boards/mbed_lpc1768/doc.txt

```diff
@@ -3,24 +3,95 @@
+See [here](https://os.mbed.com/platforms/mbed-LPC1768/) for more information
+about the board.
 
-*missing*
+## Hardware
 
-# Known Issues
+### MCU
+
+| MCU             | LPC1768       |
+|:--------------- |:------------- |
+| Family          | ARM Cortex-M3 |
+| Vendor          | NXP           |
+| RAM             | 64kB SRAM     |
+| Flash           | 512kB         |
+| Frequency       | up to 100MHz  |
+| UARTs           | 3             |
+| SPIs            | 2             |
+| I2Cs            | 2             |
+| CAN             | 2             |
+| PWM             | 6             |
+| USB Host/Device | 1             |
+| Ethernet        | 1             |
+| ADC             | 6             |
+| Datasheet       | [Datasheet](http://www.nxp.com/documents/data_sheet/LPC1769_68_67_66_65_64_63.pdf) |
+| User Manual     | [User Manual](http://www.nxp.com/documents/user_manual/UM10360.pdf)|
+
+### User Interface
+
+1 Button:
+
+| NAME   |  SW1  |
+|:-----  |:----- |
+| Pin    | RESET |
+
+5 LEDs:
+
+| NAME  | LED1  | LED2  | LED3  | LED4  | STATUS |
+| ----- | ----- | ----- | ----- | ----- | ------ |
+| Color | blue  | blue  | blue  | blue  | blue   |
+| Pin   | P1.18 | P1.20 | P1.21 | P1.23 | N/A    |
+
+Status LED is not controlled by the LPC1768.
+
+## Implementation Status
+
+| Device           | ID              | Supported | Comments               |
+|:---------------- |:--------------- |:--------- |:---------------------- |
+| MCU              | lpc1768         | yes       |                        |
+| Low-level driver | GPIO            | yes       |                        |
+|                  | ADC             | no        |                        |
+|                  | PWM             | no        |                        |
+|                  | UART            | yes       |                        |
+|                  | I2C             | no        |                        |
+|                  | SPI             | no        |                        |
+|                  | USB             | no        |                        |
+|                  | Ethernet        | no        |                        |
+|                  | RTT             | no        |                        |
+|                  | RTC             | no        |                        |
+|                  | Timer           | yes       |                        |
+
+## Using UART
+
+This board opens a serial interface through the USB automatically.
+
+## Supported Toolchains
+
+For using the mbed LPC1768 board we strongly recommend the usage of the
+[GNU Tools for ARM Embedded Processors](https://launchpad.net/gcc-arm-embedded)
+toolchain.
+
+## Flashing the device
+
+The easiest way to flash the device is to connect to the board via USB. The
+board appears as a 2MB usb drive. Simply copy your binary file to the root
+directory and reset your board.
+While the LPC1768 is flashed by the MBED-IF01 with the new firmware, the
+status LED blinks fast.
```

Comment:
I suppose `make flash` still works? (IMHO that's much easier workflow-wise than copying files)

Reply:
Yes `make flash` still works and its sole purpose is to copy the binary file to the USB flash drive. I also mentioned that the user needs to press the reset button to start the new firmware.